### DISCUSSION TITLE
章节标题中间的空格有办法消去吗？

### DISCUSSION THREAD

**Discussion by sastars9** [nju-lug/NJUThesis#190] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/128053942/227724750-001a0b87-6bd6-40fd-9392-f893fd38d387.png)
wo'jia我将章节名修改成这样，能不能把空格也删掉呢？

**Comment by note286** [comment]
你咋修改的。

**Reply by sastars9** [reply]
`\ctexset{
  subsection/name = {,、},
  subsection/number = \chinese{subsection}
  }`在导言区

**Comment by note286** [comment, marked as answer]
文档要看全。

```latex
\documentclass{njuthesis}
\ctexset{subsection/name={,、},subsection/number=\chinese{subsection},subsection/aftername={}}
\begin{document}
\chapter{测试}
\section{测试}
\subsection{测试}
\end{document}
```

![image](https://user-images.githubusercontent.com/16207292/227756518-16364dbb-79fa-42ec-933c-f30df000b897.png)

**Reply by sastars9** [reply]
感谢，好用
